include/libdnf5/conf/config_main.hpp

```diff
@@ -44,8 +44,6 @@ class LIBDNF_API ConfigMain : public Config {
 
     OptionNumber<std::int32_t> & get_debuglevel_option();
     const OptionNumber<std::int32_t> & get_debuglevel_option() const;
-    OptionNumber<std::int32_t> & get_errorlevel_option();
```

Comment:
This breaks API. You should bump a major version of the project and a soname of libdnf.

Reply:
Indeed, I would probably do that together with some more significant changes.
If you agree, I made a PR to revert the breakage: https://github.com/rpm-software-management/dnf5/pull/1793.

Reply:
Oops, that didn't occur to me. Thanks for noticing that, Petr.

Reply:
That's also an option.